src/WobblySpinner.props.js

```diff
@@ -0,0 +1,21 @@
+import { PropTypes } from "react";
+
+export const propTypes = {
```

Comment:
I just found the size of the file larger than it needed to be. I like the pattern of just looking at the code, not the stuff that is less likely to change.